examples/simple_example/main.tf

```
@@ -62,7 +62,7 @@ module "dataflow-bucket" {
 
 module "dataflow-job" {
   source  = "terraform-google-modules/dataflow/google"
```

This would require root module to be present. The lint is failing because root module doesn't exists. Should it refer to legacy module? let's do it everywhere wherever root module is used.

```suggestion
  source  = "terraform-google-modules/dataflow/google//modules/legacy"
```